### PR TITLE
feat: add yuck

### PR DESCRIPTION
**Pull request by Philipp-M**
Adds support for yuck (eww).

### CODE REVIEW

**Comment by amaanq** [comment]
Related: https://github.com/Philipp-M/tree-sitter-yuck/pull/1

File: queries/yuck/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,66 @@
+(ERROR) @error
+
+(line_comment) @comment
+
+; keywords and symbols
+
+(keyword) @keyword
+(symbol) @function
```
Comment (theHamsta) [inline queries/yuck/highlights.scm:8]: function is probably correct here (I don't know the language). Just wanted to make aware of `@symbol`

File: queries/yuck/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+; literals
+
+(bool_literal) @constant.builtin.boolean
```
Comment (theHamsta) [inline queries/yuck/highlights.scm:12]: ` @boolean`. Do other editors use `@constant.builtin.boolean`?

**Review by theHamsta — APPROVED** [review]

**Comment by amaanq** [comment]
Yeah sorry, we're working out kinks in the PR I submitted to improve the repo/grammar, I got behind on that so I'll lyk when that's ready

File: queries/yuck/highlights.scm (hunk 1 above)
Comment (Philipp-M) [reply queries/yuck/highlights.scm:8]: Well actually I used `@tag` in helix. `symbol` is most of the time *some* kind of DOM/graphical element, like `box` or `defwidget` etc. but not all the time, there is e.g. a `for` symbol. So I don't know maybe also use `@tag`, I'm not sure?

File: queries/yuck/highlights.scm (hunk 2 above)
Comment (Philipp-M) [reply queries/yuck/highlights.scm:12]: Well I guess this is yet another tag that differs between helix (my primary editor currently) and neovim.

**Comment by Philipp-M** [comment]
This should now be ready to review/merge, the linked PR has been merged.

**Comment by amaanq** [comment]
Last CI is stuck for no reason, seems good to merge

**Pull request merged**